docs/source/guides/libraries/swift_ios.rst

```diff
@@ -0,0 +1,282 @@
+iOS Swift Library
+-----------------
+
+Objectives
+^^^^^^^^^^
+
+In this guide, you will learn:
+
+-	How to build client library for iOS
+-	How to configure test application
+-	How to interact with Iroha blockchain from the mobile device
+
+Video Guide
+^^^^^^^^^^^
+
+For more details please visit the video below which fully describes all the steps.
+
+.. raw:: html
+
+    <iframe width="560" height="315" style="margin-bottom: 20px" src="https://www.youtube.com/embed/sjuK3I1I080" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+
+Prerequisites
+^^^^^^^^^^^^^
+
+Before starting you need to install the following software on your mac:
+
+-	XCode
+-	Carthage
+-	Git
+-	Cmake
+-	Postgresql
+
+This tutorial was tested with the following environment:
+
+-	MacOS Sierra 10.12.6
+-	Xcode 9.2
+-	carthage 0.29.0
+-	cmake 3.11.0
+-	iPhone 7 iOS 11.2 Simulator
+ 
+Hyperledger Iroha iOS library
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Iroha has the following features:
+
+1.	Creation and management of custom complex assets, such as currencies or indivisible rights, serial numbers, patents, etc.
+2.	Management of user accounts
+3.	Taxonomy of accounts based on domains — or sub-ledgers in the system
+4.	The system of rights and verification of user permissions for the execution of transactions and queries in the system
+5.	Validation of business rules for transactions and queries in the system
+
+Among the non-functional requirements can be noted a high degree of network fault tolerance (Byzantine Fault Tolerant).
+Iroha iOS library gives the ability to provide key generation and signing logic for queries and transactions passed to Iroha blockchain.
+Let's start with the detailed instructions how to install Iroha on the local machine.
+
+
+ 
+Instruction
+^^^^^^^^^^^
+
+1.	Open the terminal and go to the folder where You want to install all artifacts:
+
+  .. code-block:: bash
+
+      cd path/to/your/folder/for/example/iroha-ios/project/
+
+2.	Clone the repository for the iOS client:
+
+  .. code-block:: bash
+
+      git clone https://github.com/soramitsu/iroha-ios.git
+
+3.	Go to the Iroha-ios folder:
+
+  .. code-block:: bash
+
+      cd iroha-ios/
+
+4.	Update dependencies:
+
+  .. code-block:: bash
+
+      carthage update --platform iOS
+
+5.	Go to sample project directory:
+
+  .. code-block:: bash
+
+      cd SwiftyIrohaExample
+
+6.	Update dependencies for the sample:
+
+  .. code-block:: bash
+
+      carthage update --platform iOS
+
+7.	Go to GRPC library source's location:
+
+  .. code-block:: bash
+
+      cd grpc-swift/
+
+8.	Remove old library sources:
+
+.. note:: Make sure You've located in ``grpc-swift/`` subfolder
```

Comment:
Prefer using non-shortened version (e.g You have)

Also "You" with capital letter seems archaic
and most probably have -> are